packages/core/src/customer/strategies/stripe-upe/stripe-upe-customer-strategy.ts

```diff
@@ -0,0 +1,87 @@
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { InvalidArgumentError, MissingDataError, MissingDataErrorType } from '../../../common/error/errors';
+import { PaymentMethodActionCreator } from '../../../payment';
+import { StripeElements, StripeElementType, StripeEvent, StripeScriptLoader, StripeUPEClient } from '../../../payment/strategies/stripe-upe';
+import CustomerActionCreator from '../../customer-action-creator';
+import CustomerCredentials from '../../customer-credentials';
+import { CustomerInitializeOptions, CustomerRequestOptions, ExecutePaymentMethodCheckoutOptions } from '../../customer-request-options';
+import CustomerStrategy from '../customer-strategy';
+
+export default class StripeUPECustomerStrategy implements CustomerStrategy {
+    private _stripeElements?: StripeElements;
+
+    constructor(
+        private _store: CheckoutStore,
+        private _stripeUPEScriptLoader: StripeScriptLoader,
+        private _customerActionCreator: CustomerActionCreator,
+        private _paymentMethodActionCreator: PaymentMethodActionCreator
+    ) {}
+
+    async initialize(options: CustomerInitializeOptions): Promise<InternalCheckoutSelectors> {
+        const { methodId, stripeupe: { container, onEmailChange } = {} } = options;
+        let stripeUPEClient: StripeUPEClient;
+
+        if (!methodId) {
```

Comment:
Do we need to check here if ` is present as well?

Reply:
Hi @animesh1987, the interface doesn't support undefined but I could assign a default value on the destructuring and delete the conditional